src/config.cpp

```diff
@@ -115,16 +118,53 @@ nlohmann::json expandVariables(const nlohmann::json& json,
     return jsonFlat.unflatten();
 }
 
+std::string getJSONValue(const nlohmann::json& json,
```

Comment:
what if a string value isn't desired? perhaps template it?

Reply:
I used only within the initialization, but in the future it could be useful as well. 
So, template parameterized, thanks!